src/Sentry/SentryEvent.cs

```diff
@@ -96,6 +96,12 @@ public sealed class SentryEvent : IEventLike, IJsonSerializable
             set => SentryThreadValues = value != null ? new SentryValues<SentryThread>(value) : null;
         }
 
+        /// <summary>
+        /// The Sentry Debug Meta Images interface.
+        /// </summary>
+        /// <see href="https://develop.sentry.dev/sdk/event-payloads/debugmeta#debug-images"/>
+        public List<SentryDebugImage>? SentryDebugImages { get; set; }
```

Comment:
Property names don't need to be prefixed with `Sentry`:

```suggestion
        public List<SentryDebugImage>? DebugImages { get; set; }
```

Reply:
Sure thing. I was following the `public IEnumerable<SentryThread>? SentryThreads` example. But maybe those are named that way to avoid conflicts with stdlib types probably?